src/sentry/templates/sentry/js-sdk-loader.ts

```diff
@@ -218,19 +222,12 @@ declare const __LOADER__IS_LAZY__: any;
 
   _window[_namespace].onLoad = function (callback) {
     onLoadCallbacks.push(callback);
-    if (lazy && !forceLoad) {
```

Comment:
I think we can get rid of this. Instead, when a user call `Sentry.forceLoad()`, we simply inject the SDK the same way we would for a non-lazy setup - all the other logic is not needed IMHO?